docs/value.md

```diff
@@ -111,10 +110,10 @@ behavior:
   prop.stopListening( observable );
   ```
 
-- __prop.lastSet__ `{can-simple-observable}` An observable value that gets set when this
-  property is set.  You can read its value or listen to when its value changes to
-  derive the property value.  The following makes `property` behave like a
-  normal object property that can be get or set:
+  - __prop.lastSet__ `{can-simple-observable}` An observable value that gets set when this
+    property is set.  You can read its value or listen to when its value changes to
+    derive the property value.  The following makes `property` behave like a
+    normal object property that can be get or set:
 
   ```js
```

Comment:
I think indentation broke here.  The code block should be indented.

![image](https://user-images.githubusercontent.com/78602/45120177-032c3280-b123-11e8-8176-5cfac31b565c.png)

Reply:
Is it possible to codepen this?

Reply:
I don't think a code block is indented anywhere in the documentation. This is now fixed as far as `property` and `value` as well as making it a codepen.